sdks/python/apache_beam/typehints/typehints.py

```diff
@@ -83,6 +82,11 @@
     'Generator',
     'WindowedValue',
     'TypeVariable',
+    'TypeConstraint',
```

Comment:
It was to fix broken cross references (since, if a class is not public, the reference won't be generated and thus cross references to it will be broken), but if we don't want to expose extra classes, I'll just add those references to the ``nitpick_ignore`` list in ``conf.py``.